### PR TITLE
misc: add readme

### PR DESCRIPTION
**Pull request by zipzap11**

### CODE REVIEW

**Review by zhenqianz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,111 @@
...
+
+func (s *JSSubscriber) RegisterNATSJetStream(js ferstream.JetStream) {
+	p.js = js
+}
+
+func (s *JSSubscriber) InitStream() error {
+	// you can do anything you need here before adding stream
+	_, err := p.js.AddStream(&nats.StreamConfig{
+	    Name: "YOUR_STREAM_NAME",
+		Subject: []string{"YOUR_STREAM_NAME.>"},
```
Comment (atjhoendz) [inline README.md:55]: SUBJECT

File: README.md [hunk 2]
```diff
@@ -1 +1,111 @@
...
+	)
+	msgHandler := func (payload ferstream.MessageParser) error {
+            msg, _ := payload.(*ferstream.NatsEventMessage)
+			// do something with the message
+            return nil
+	}
+	errHandler := func (payload ferstream.MessageParser) error {
+		// something you want to do if the msgHandler returns an error
+	}
+	s.js.QueueSubscribe("YOUR_STREAM_NAME.>", "your-queue-group",
```
Comment (atjhoendz) [inline README.md:77]: first param = SUBJECT

File: README.md [hunk 3]
```diff
@@ -1 +1,111 @@
...
+            return nil
+	}
+	errHandler := func (payload ferstream.MessageParser) error {
+		// something you want to do if the msgHandler returns an error
+	}
+	s.js.QueueSubscribe("YOUR_STREAM_NAME.>", "your-queue-group", 
+	    _, err := ferstream.NewNATSMessageHandler(new(ferstream.NatsEventMessage), retryAttempts, retryInterval, msgHandler, errHandler)
+            if err != nil {
+                // handle err
+            }
```
Comment (atjhoendz) [inline README.md:81]: kurang config manualAck sama durable nya jung

**Review by atjhoendz — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,113 @@
...
+
+func (p *JSPublisher) RegisterNATSJetStream(js ferstream.JetStream) {
+	p.js = js
+}
+
+func (p *JSPublisher) InitStream() error {
+	// you can do anything you need here before adding stream
+	_, err := p.js.AddStream(&nats.StreamConfig{
+	    Name: "YOUR_STREAM_NAME",
+		Subject: []string{"YOUR_STREAM_NAME.>"},
```
Comment (atjhoendz) [inline README.md:32]: SUBJECT.>

**Review by atjhoendz — APPROVED** [review]

**Pull request merged**